### PR TITLE
JFrog.Conan version 1.39.0

### PR DESCRIPTION
**Pull request by KaranKad**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22559)

### CODE REVIEW

File: manifests/j/JFrog/Conan/1.39.0/JFrog.Conan.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  InstallerType: inno
+  InstallerUrl: https://github.com/conan-io/conan/releases/download/1.39.0/conan-win-64.exe
+  InstallerSha256: 59D0B0B3F4418CD7846160A00FF2E1F2C2E5C6C665836D52954C1C25AF213D73
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/conan-io/conan/releases/download/1.39.0/conan-win-32.exe
+  InstallerSha256: CE3DBB37465EB6190B0AA7DA66EE307D9A413BEA59A3EDF49D252EFB8FDA1ECE
+  UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/j/JFrog/Conan/1.39.0/JFrog.Conan.installer.yaml:23]: * Added ProductCode sinde the package is missing Publisher
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget list
Name                                    Id                                           Version           Available   Source
-------------------------------------------------------------------------------------------------------------------------
Conan Package Manager version 1.39.0    Conan Package Manager_is1                    1.39.0
```
```suggestion
  UpgradeBehavior: install
  ProductCode: 'Conan Package Manager_is1'
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: inno
  InstallerUrl: https://github.com/conan-io/conan/releases/download/1.39.0/conan-win-32.exe
  InstallerSha256: CE3DBB37465EB6190B0AA7DA66EE307D9A413BEA59A3EDF49D252EFB8FDA1ECE
  UpgradeBehavior: install
  ProductCode: 'Conan Package Manager_is1'
```

File: manifests/j/JFrog/Conan/1.39.0/JFrog.Conan.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+PackageIdentifier: JFrog.Conan
+PackageVersion: 1.39.0
+PackageLocale: en-US
+Publisher: JFrog
+PublisherUrl: https://conan.io/
+PrivacyUrl: https://conan.io/privacy-policy.html
+PackageName: Conan
+PackageUrl: https://conan.io/
+License: MIT
+LicenseUrl: https://raw.githubusercontent.com/conan-io/conan/develop/LICENSE.md
```
Comment (OfficialEsco) [inline manifests/j/JFrog/Conan/1.39.0/JFrog.Conan.locale.en-US.yaml:13]: * Changed PackageName to "Conan Package Manager"
* Added PublisherSupportUrl
* Added Copyright and Copyright Url
```suggestion
Publisher: JFrog
PublisherUrl: https://conan.io
PublisherSupportUrl: https://github.com/conan-io/conan/issues
PrivacyUrl: https://conan.io/privacy-policy.html
PackageName: Conan Package Manager
PackageUrl: https://conan.io
License: MIT
LicenseUrl: https://raw.githubusercontent.com/conan-io/conan/develop/LICENSE.md
Copyright: Copyright (c) 2019 JFrog LTD
CopyrightUrl: https://raw.githubusercontent.com/conan-io/conan/develop/LICENSE.md
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**